wiki/content/mutations/index.md

```diff
@@ -923,40 +920,24 @@ Result:
       "uid(v)": "0x2"
     }
   },
-  "extensions": {
-    "server_latency": {
-      "parsing_ns": 71033,
-      "processing_ns": 22994018
-    },
-    "txn": {
-      "start_ts": 4,
-      "commit_ts": 5,
-      "preds": [
-        "1-email",
-        "1-name"
-      ]
-    }
-  }
+  "extensions": {...}
 }
 ```
 
-The upsert first checks whether a user with the email `user@dgraph.io` exists. Because
-the database is currently empty, no such user exists. Hence, the variable `v` will be
-empty. In this case, the `uid` function returns a new UID for the variable `v` replacing
-with the new UID value wherever `uid(v)` is used.
-
-### Update Use Case
+The query part of the upsert block stores the UID of the user with the provided email
+id in the variable `v`. The mutation part then extracts the UID from variable `v` and
+stores the `name` and `email` information in the database. In case the user exists,
+the information is updated whereas if the user doesn't exist, `uid(v)` is treated
+as a blank node and a new user is created as explained above.
 
-Now, we want to add the `age` information for the same user having email ID
-`user@dgraph.io`. We can use the Upsert block to do the same as follows:
+Now, we want to add the `age` information for the same user having the same email ID
+`user@company1.io`. We can use the Upsert block to do the same as follows:
 
 ```sh
 curl -H "Content-Type: application/rdf" -X POST localhost:8080/mutate?commitNow=true -d  $'
 upsert {
   query {
-    me(func: eq(email, "user@dgraph.io")) {
-      v as uid
-    }
+    v as var(func: eq(email, "user@company1.io"))
   }
 
   mutation {
```

Comment:
This mutation would create a user without a name ... not sure this is a good example.

Reply:
Maybe it's worth pointing to the conditional upserts here?